### PR TITLE
Bug 1221041 - fixes example in osc env

### PR DESCRIPTION
**Pull request by fabianofranz**

### CODE REVIEW

**Comment by fabianofranz** [comment]
[merge]

File: pkg/cmd/cli/cmd/env.go [hunk 1]
```diff
@@ -63,8 +63,8 @@ syntax.`
   // Update all replication controllers in the project to have ENV=prod
   $ %[1]s env replicationControllers --all ENV=prod
 
-  // Remove the enviroment variable ENV from a pod definition on disk and update the pod on the server
-  $ %[1]s env -f pod.json ENV-
+  // Remove the enviroment variable ENV from all deployment configs
```
Comment (smarterclayton) [inline pkg/cmd/cli/cmd/env.go]: Why did you change this?

Comment (fabianofranz) [inline pkg/cmd/cli/cmd/env.go]: Should not, or maybe make it an addition? 

Seemed a more intuitive example of removal, since removing based on a definition on file does not work in some cases. You can't reproduce that using the sample-app processed files, for example.

----- Original Message -----

Comment (smarterclayton) [inline pkg/cmd/cli/cmd/env.go]: The example is demonstrating a use case where you are using a local copy of config to apply to the server.  It's still relevant.

----- Original Message -----

Comment (fabianofranz) [inline pkg/cmd/cli/cmd/env.go]: I'll make my example an addition then, tks.

----- Original Message -----
